codeuino/Codebadge
# Modified Login UI

### About
This PR modifies the default authorization login user-interface. It also adds the ability to logout once logged in by deleting the saved access token.
_____________________________________________
### Changes
- Extracted the toolbar to a new component
- Changed the whole look of the authorization page
- Added a logout feature to the toolbar which deletes the stored access token and allows you to login again.
_____________________________________________
### Expected Results
The authorization page should look like this:
![expected](https://user-images.githubusercontent.com/41499871/70819483-60219880-1dfc-11ea-8c1c-7564be0ed3b9.png)


The Navigation Bar should look like this after logging in:
![expected2](https://user-images.githubusercontent.com/41499871/70818233-387d0100-1df9-11ea-92d2-7fde27eeb364.png)
______________________________________________
### Testing
- Launch your application locally by follow the steps in the [readme](https://github.com/codeuino/Codebadge/blob/master/README.md)
- Make sure the page looks similar to the one as described above.
- Click on the Login button.
- After logging in make sure the Logout button appears on the navigation bar.
_______________________________________________
